pages/installation/os/sles.rst

```diff
@@ -0,0 +1,122 @@
+*******************
+SLES installation
+*******************
+
+This guide describes the fastest way to install Graylog on SLES 12 SP3. All links and packages are present at the time of writing but might need to be updated later on.
+
+.. warning:: This setup should not be done on publicly exposed servers. This guide **does not cover** security settings!
+
+
+Prerequisites
+-------------
+
+The following Patterns are use as a minimal server setup::
+
+  - Base System
+  - Minimal System (Appliances)
+  - Yast configuration packages
+
+.. warning:: This Guide makes the assumption that the firewall is disabled and communication is possible to the outside world.
+
+Taking a minimal server setup as base will need this additional packages::
+
+  $ sudo zypper install java-1_8_0-openjdk
+
+
+MongoDB
+-------
+
+Installing MongoDB on SLES should follow `the tutorial for SLES <https://docs.mongodb.com/v3.4/tutorial/install-mongodb-on-suse/>`_ from the MongoDB documentation. Add the GPG Key and the Repository before you install MongoDB::
+
+  $ sudo rpm --import https://www.mongodb.org/static/pgp/server-3.4.asc
+  $ sudo zypper addrepo --gpgcheck "https://repo.mongodb.org/zypper/suse/12/mongodb-org/3.4/x86_64/" mongodb
+  $ sudo zypper -n install mongodb-org
+
+
+
+Additionally, run these last steps to start MongoDB during the operating system's boot and start it right away::
```

Comment:
```
In order to automatically start MongoDB on system boot, you have to activate the MongoDB service by running the following commands::
```

Reply:
I'm not 100% sure if this is different on SUSE Linux, but on any other systemd-based distro, it's sufficient to run the following command to start services on boot:
```
sudo systemctl enable mongod.service
```

Reply:
https://www.suse.com/documentation/sles-12/singlehtml/book_sle_admin/book_sle_admin.html#sec.boot.systemd.basics.services_enabling.cmd says using the `systemctl` command should suffice (for systemd-based services).

Reply:
I took the "what todo" from the MongoDB Documentation 
https://docs.mongodb.com/v3.4/tutorial/install-mongodb-on-suse/#verify-that-mongodb-has-started-successfully

This was the only way how I was able to have MongoDB reboot safe.

Reply:
✅